test/test-pages.js

```diff
@@ -17,27 +17,22 @@ it('Landing page status', function (done) {
 });
 
 it('Dashboard categories page', function (done) {
-  request('http://localhost:3000/dashboard/testorg/categories', function (error, response, body) {
-    expect(response.statusCode).to.equal(200);
-    expect(body).to.contain("testorg's Dashboard");
-    expect(body).to.contain("Add a category");
+  request('http://localhost:3000/dashboard/categories', function (error, response, body) {
+    expect(response.statusCode).to.equal(401);
```

Comment:
Are we planning on changing this later so test can load the actual page? Should add TODO comments then.

Reply:
I don't think this comment was resolved.

Reply:
Fixed with comment [comment](https://github.com/googleinterns/donatia/pull/55#discussion_r452351701)

Reply:
Oh I see. I was commenting on how the page now loads a 401 error. Are we planning on eventually loading the page somehow?